examples/simple_plugin/EntityManagementFeatures.hh

```diff
@@ -17,11 +17,11 @@
 
 //! [basic include]
 #include <string>
-#include <ignition/physics/Implements.hh>
+#include <gz/physics/Implements.hh>
 //! [basic include]
 
 //! [include feature]
-#include <ignition/physics/ConstructEmpty.hh>
+#include <gz/physics/ConstructEmpty.hh>
 
 namespace ignition {
```

Comment:
The `ign-physics2` branch uses `gz` instead of `ignition` here, but that failing to build locally, so I changed it back to `ignition`. I suspect `gz/physics/config.hh` is not being included the same way it is in `ign-physics2`, but I'm not sure where.